modules/api/src/main/java/org/apache/ignite/configuration/schemas/table/TableConfigurationSchema.java

```diff
@@ -35,6 +36,12 @@ public class TableConfigurationSchema {
     @InjectedName
     public String name;
 
+    /** Integer table id. */
+    @Immutable
+    @Range(min = 1, max = Integer.MAX_VALUE)
+    @Value(hasDefault = true)
+    public int intId = 1;
```

Comment:
Why such a name? seems to be more correct just **id**.
Why such a range?
Can we use UUID as an identifier?

Reply:
UUID is already a main "globally unique" identifier. This one - I'm not sure about the name, honestly. Field named "id" already exists.